ethcore/src/externalities.rs

```diff
@@ -395,11 +395,11 @@ impl<'a, T: 'a, V: 'a, B: 'a> Ext for Externalities<'a, T, V, B>
 	}
 
 	fn add_sstore_refund(&mut self, value: U256) {
-		self.substate.sstore_clears_refund = self.substate.sstore_clears_refund.saturating_add(value);
```

Comment:
We are loosing the saturating property here, maybe it could be saturating operation in I256 implementation by default (we need to renaim I256 to make it clear that it does not behave as U256)

Reply:
I few nit, maybe also some super basic arithmetic tests case on I256 would be welcome.